templates/_macros/hero.jinja

```diff
@@ -0,0 +1,109 @@
+{% from "_macros/cta/cta-block.jinja" import cta_block %}
+
+# Params
+# title_text: Hero title text (required)
+# subtitle_text: Hero subtitle text
+# is_full_width_on_tablet: whether image is in its own row on tablet.
+#   If false, image will be in its own row (stacked below header/description)
+# image_src: image source URL (required)
+# image_alt: image alt text
+# image_aspect_ratio: aspect ratio identifier of the image, from the `p-image-container` component
+# image_full_cover: whether the image should span the full width of the bottom of the hero section
+# image_hidden_breakpoints: list of breakpoints where the image should be hidden, e.g. ["large", "medium"] to only show on small screens
+# show_cta: whether to show the call-to-action block
+# cta_primary_text: primary CTA button text, shown in a primary-colored button
+# cta_primary_url: primary CTA button URL
+# cta_secondary_text: secondary CTA button text, shown as a link
+# cta_secondary_url: secondary CTA button URL
+# Slots
+# description: slot for additional paragraph-style content below the title and subtitle
+
+{% macro hero(
+  title_text="",
+  subtitle_text="",
+  is_full_width_on_tablet = true,
+  image_src = "",
+  image_alt = "",
+  image_aspect_ratio = "",
+  image_full_cover = false,
+  image_hidden_breakpoints = [],
```

Comment:
I suppose we could accept device names instead of breakpoint names, and then have some mapping in the macro, like this

| Device identifier | Breakpoint identifier |
|-------------------|-----------------------|
| mobile            | small                 |
| tablet            | medium                |
| desktop           | large                 |


Then, input such as this could be used to hide the image on "mobile" size (small breakpoint):

```html
{% call(slot) hero(
  image_hidden_on_sizes=["mobile"]
) %}
```

Thoughts?

Reply:
This has been added to the PR, let me know your thoughts

Reply:
Hidden image breakpoints have been removed from the PR entirely per Bartek's suggestion; users now have responsibility to pass in image content with whatever classes they need